project/legislation/propublica/congress/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""
+A Python client for the ProPublica Congress API
+
+API docs: https://propublica.github.io/congress-api-docs/
+
+Most of my code has been either influenced by, or replaced by
+code from Chris Amico's Propublica API code.
+"""
+import os
+
+from .client import Client
+from .utils import CongressError, NotFound, check_chamber, get_congress, CURRENT_CONGRESS
+
+# subclients
+from .bills import BillsClient
+from .members import MembersClient
+from .committees import CommitteesClient
+from .votes import VotesClient
+from .nominations import NominationsClient
+
+
+__all__ = ('Congress', 'CongressError', 'NotFound', 'get_congress', 'CURRENT_CONGRESS')
+
+
+class Congress(Client):
```

Comment:
Please move this class to a file named **congress.py**. It does not seem conventional to define classes in an `__init__.py' file, so may be a bit confusing when reading/maintaining this project's codebase.